documentation/polymer-templates/tutorial-template-basic.asciidoc

```diff
@@ -1,24 +1,55 @@
 ---
-title: Creating a Simple Component Using the PolymerTemplate API
+title: Creating a Simple Component Using the Template API
 order: 2
 layout: page
 ---
 
-ifdef::env-github[:outfilesuffix: .asciidoc]
+= Creating a Simple Component Using the Template API
 
-= Creating a Simple Component Using the PolymerTemplate API
+In this section, we demonstrate how to create a simple component using only the Template API.
 
-In this section, we demonstrate how to create a simple component using only the `PolymerTemplate` API.
+It's recommended to use LitElement template language for your components, but PolymerTemplate is also supported.
```

Comment:
Contraction "It's" perhaps a tad too informal. Also, "It is" seems to have been preferred throughout:
```
~/vaadin/flow-and-components-documentation }> grep -r "It is" documentation | wc -l 
      76
~/vaadin/flow-and-components-documentation }> grep -r "It's" documentation | wc -l   
       4
```

Reply:
Good comment. Fixed and I also verified all other pages.